doc/whatsnew/3/3.2/index.rst

```diff
@@ -14,6 +14,41 @@ Summary -- Release highlights
 
 .. towncrier release notes start
 
+What's new in Pylint 3.2.4?
+---------------------------
+Release date: 2024-06-24
```

Comment:
I can approve today if you would like to bump this line.

Reply:
```suggestion
Release date: 2024-06-25
```